### PR TITLE
Issue 124 update naturalgas to fossilgas try2

### LINKED ISSUE #124: Update References to "Natural Gas" to "Gas" or "Methane Gas"
Natural Gas is a marketing and not technical term, so we should update it to something a little more accurate. Let's change it to "Fossil Gas".

### PR DESCRIPTION
**Pull request by sburnschicago**
# Description
Hi Victor, I ended up creating a new branch to start with a more recent version of the code. First branch can be deleted?
This one has no changes to the data source, just changes to the static site where Natural Gas was mentioned.

Took another effort at editing the NaturalGasUse as it comes from the data source and displays on the site, but seems to be beyond me. In common_functions.vue was a start on it.

Seems that if the data source can't be changed, then there would need to be a transformation step in between, but only for where the data is displayed?

Resolves #124

# Testing Instructions

Please describe the tests/QA that you did to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
Didn't seem necessary - only word change on the front-end
However, for transformation steps, documentation sounds great.

- [x] I have added tests that prove my fix is effective or that my feature works
Website loads fine.

- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/common-functions.vue [hunk 1]
```diff
@@ -264,4 +263,12 @@ export function estimateUtilitySpend(energyUseKbtu: number, isElectric: boolean)
     return Math.round(estimateRaw / 10) * 10;
   }
 }
-</script>
+
+export function transformBuildingData(building: IBuilding): IBuilding {
```
Comment (vkoves) [inline src/common-functions.vue:267]: Let's remove the changes to this file if they aren't needed

**Review by sburnschicago — COMMENTED** [review]
just removing one character that was confusing, but was in a comment, so, not a big deal.

**Comment by sburnschicago** [comment]
Ok, I think we're all good, except for the static images in-power-chart.png and .webp in electrify-chicago/static. They refer to Natural gas in the graph, on the About page.

**Comment by sburnschicago** [comment]
Tests performed were building the site, going to every page, manually searching for Natural. Also loading some building and seeing if they loaded the correct replacement Fossil for Natural.

**Review by vkoves — APPROVED** [review]
Looks great, awesome work! Did a quick spot check myself, and the code is super simple and clean 👌

**Pull request merged**